docs/index.rst

```diff
@@ -1,21 +1,20 @@
 .. -*- coding: utf-8 -*-
 
-====================================
-Process for Plone core's development
-====================================
+=========================
+Developing for Plone Core
+=========================
 
 Introduction
 ============
 This documentation describes the process for developing Plone.
 It is primarily a technical resource for setting up your core developer buildout,
-fixing bugs,
-and writing plips.
+fixing bugs and writing plips.
 
 STOP!
```

Comment:
@svx *sorry for hijacking your changes*

Can we, pretty please, rephrase this header as well? There's nothing more discouraging than yelling "STOP!"...

Something like *A word of warning* would be probably less discouraging (other suggestions welcome).